documentation/docs/scenarios/ui5-sap-cp/Readme.md

```diff
@@ -17,21 +14,28 @@ This document describes a scenario step, which means that it combines various di
 
 ### Project Prerequisites
 
-This scenario step requires additional files in your project and the execution environment on your Jenkins instance. On the project level, provide and adjust the following template:
+This scenario requires additional files in your project and in the execution environment on your Jenkins instance. 
+
 
-| File Name | Description |
-|-----|-----|
-| [`.npmrc`](https://github.com/marcusholl/jenkins-library/tree/pr/scenarioUI5SAPCP/documentation/docs/scenarios/ui5-sap-cp/files/.npmrc) | This file contains a reference to the SAP NPM registry (`@sap:registry https://npm.sap.com`), which is required to fetch dependencies to build the application. Place it in the root directory of your project. |
-| [`mta.yaml`](https://github.com/marcusholl/jenkins-library/tree/pr/scenarioUI5SAPCP/documentation/docs/scenarios/ui5-sap-cp/files/mta.yaml) | This file controls the behavior of the MTA toolset. Place it in your application root folder and adjust the values in brackets with your data. |
-| [`package.json`](https://github.com/marcusholl/jenkins-library/tree/pr/scenarioUI5SAPCP/documentation/docs/scenarios/ui5-sap-cp/files/package.json) | This file lists the required development dependencies for the build. Add the content to your existing `package.json` file. |
-| [`Gruntfile.js`](https://github.com/marcusholl/jenkins-library/tree/pr/scenarioUI5SAPCP/documentation/docs/scenarios/ui5-sap-cp/files/Gruntfile.js) | This file controls the grunt build. By default the tasks `clean`, `build`, and `lint` are executed. Place it in the root directory of your project. |
+On the project level, provide and adjust the following template:
+
+| File Name | Description | Position |
+|-----|-----|-----|
+| [`.npmrc`](https://github.com/marcusholl/jenkins-library/tree/pr/scenarioUI5SAPCP/documentation/docs/scenarios/ui5-sap-cp/files/.npmrc) | This file contains a reference to the SAP NPM registry (`@sap:registry https://npm.sap.com`), which is required to fetch the dependencies required to build the application. | Place the `.npmrc` file in the root directory of your project. |
```

Comment:
Ups, it is wrong to denote "marcusholl" here. Was wrong before that change already.
Applies also elsewhere in this file

Reply:
See #489

Reply:
I've exchanged the links - can you please tell me how to proceed?

Reply:
> can you please tell me how to proceed?

What exactly is the question? I would say: merge it.

I provided the other PR #489 since the wrong URL are basically completely independent from your PR, but it is also fine to fix it like this. On the formal level it might be better to handle the other PR independent from this one, but let's be pragmatic here ...

When this one gets merged I will close the other one.

Reply:
My question was how to proceed with the other pull request - which you've answered. I'll bear the formal level in mind for the next time.